dev/tasks/python-wheels/github.windows.yml

```diff
@@ -56,7 +56,7 @@ jobs:
           if errorlevel 1 (
             archery docker build --no-pull python-wheel-windows-vs2019 || exit /B 1
           )
-          archery docker run --no-build -e SETUPTOOLS_SCM_PRETEND_VERSION={{ arrow.no_rc_version }} python-wheel-windows-vs2019
+          archery docker run -e SETUPTOOLS_SCM_PRETEND_VERSION={{ arrow.no_rc_version }} python-wheel-windows-vs2019
```

Comment:
Should we revert this line and update the pre-built docker image instead? BTW, this is ready for review now. @raulcd @kou

Reply:
Yes. Please revert this for now.
Could you update https://github.com/apache/arrow/blob/4cce72033cae666d22110ea4b8f32a11915214d6/.env#L97-L101 instead?

Reply:
What value should I set? `PYTHON_WHEEL_WINDOWS_IMAGE_REVISION=2024-03-19` ?

Reply:
Yes. We can use today to avoid conflicting existing revision.